File: ODOT.ARMS.Web/ClientApp/src/app/project/models/project-header.ts
export class ProjectHeader {
  projId: string;
  projectAltId: number;
  projectTitleTxt: string;
  projectType: string;
  rfpNum: string;
  agreementNum: string;
  pidNum: string;
}


export enum ProjectTypesEnum {
  STANDARD = 72,
  ROC = 73,
  ORIL = 74,
  POOLED = 75
}

File: ODOT.ARMS.Web/DTOs/ProjectHeader.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace ODOT.ARMS.Web.DTOs
{
    public class ProjectHeader
    {
        public string ProjectTitleTxt { get; set; }
        public string RfpNum { get; set; }
        public string AgreementNum { get; set; }
        public string PidNum { get; set; }
        public Guid ProjId { get; set; }
        public int ProjectAltId { get; set; }
        public string ProjectType { get; set; }
    }
}
